### PR TITLE
chore: refactor to use showApiAuthAcm from API Category

### PR DESCRIPTION
**Pull request by alharris-at**
#### Description of changes
Breaking direct dependencies from Amplify CLI on transformer packages. There's a bit more to do, but this addresses auth and one of the core packages.

#### Issue #, if available
N/A

#### Description of how you validated changes
Tests/E2E Tests

#### Checklist
- [x] PR description included
- [x] `yarn test` passes
- [x] Tests are [changed or added](https://github.com/aws-amplify/amplify-cli/blob/dev/CONTRIBUTING.md#tests)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by danielleadams** [comment]
@alharris-at is this PR still on hold?

**Review by phani-srikar — APPROVED** [review]

**Review by dpilch — APPROVED** [review]

**Review by dpilch — APPROVED** [review]

File: packages/amplify-cli/package.json [hunk 1]
```diff
@@ -65,8 +65,6 @@
     "@aws-amplify/amplify-util-import": "2.6.0",
     "@aws-amplify/amplify-util-mock": "5.2.3",
     "@aws-amplify/amplify-util-uibuilder": "1.8.3",
-    "@aws-amplify/graphql-auth-transformer": "^2.1.5",
-    "@aws-amplify/graphql-transformer-core": "^1.3.1",
```
Comment (sobolk) [inline packages/amplify-cli/package.json:69]: lovely !

File: packages/amplify-cli/src/__tests__/commands/status.test.ts [hunk 2]
```diff
@@ -1,14 +1,12 @@
 import { stateManager, pathManager } from '@aws-amplify/amplify-cli-core';
-import { readProjectSchema } from 'graphql-transformer-core';
+import { showApiAuthAcm } from '@aws-amplify/amplify-category-api';
```
Comment (sobolk) [inline packages/amplify-cli/src/__tests__/commands/status.test.ts:2]: what's `Acm` ? public apis should avoid acronyms of possible.
(for the record - it seems this has shipped already)

File: packages/amplify-cli/src/extensions/amplify-helpers/current-cloud-backend-utils.ts [hunk 3]
```diff
@@ -1,9 +1,8 @@
 import { $TSAny, $TSContext, AmplifyFault, spinner, stateManager } from '@aws-amplify/amplify-cli-core';
 import sequential from 'promise-sequential';
-import { notifyFieldAuthSecurityChange, notifyListQuerySecurityChange, notifySecurityEnhancement } from './auth-notifications';
 import { getProviderPlugins } from './get-provider-plugins';
 import { getProjectConfig } from './get-project-config';
-
+import { checkForcedUpdates as checkApiForcedUpdates } from '@aws-amplify/amplify-category-api';
```
Comment (sobolk) [inline packages/amplify-cli/src/extensions/amplify-helpers/current-cloud-backend-utils.ts:5]: thought for future. if we ever get to `ApiCategory implements ICategory` then there should be `ICategory.checkForcedUpdate` with default no op impl. or some cross cutting mechanism.

**Review by sobolk — APPROVED** [review]
Looks good thank you!

**Pull request merged**